### ISSUE TITLE
Edge case: "Can't find variable: fileName"

### ISSUE THREAD

**Issue by drinking-code** [oven-sh/bun#1333]
### Version

0.2.0

### Platform

Darwin redacted.local 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:19:52 PDT 2022; root:xnu-8020.140.49~2/RELEASE_ARM64_T6000 arm64

### What steps will reproduce the bug?

This is a weird one:
- define a variable `fileName`
- use the variable

```javascript
const fileName = 'myValue'
console.log(fileName)
```

### How often does it reproduce? Is there a required condition?

This will only throw if the variable name is exactly "fileName".
"filename" will work, and variable names containing "fileName" + other characters also will not throw.

### What is the expected behavior?

The code above does not throw.

### What do you see instead?

<img width="508" alt="Bildschirmfoto 2022-10-16 um 19 39 30" src="https://user-images.githubusercontent.com/58665953/196049945-0e9ca4ca-04b8-4eff-b0a5-735af93524d4.png">

**Comment by Jarred-Sumner** [comment]
very strange! thanks for reporting

**Comment by xiaoxigua-1** [comment]
If the variable name `JSX` and `jsx` have the same issue.

**Comment by Electroid** [comment]
This has now been fixed as of `0.2.2`.

```js
const JSX = 'hello'
const fileName = 'hello2'
console.log(JSX) // "hello"
console.log(fileName) // "hello2"
```

**Comment by Electroid** [comment]
This was fixed, tested as of Bun v1.0.7.
